ee/migrations/0005_copy_over_non_ee_definitions_tables.py

```diff
@@ -0,0 +1,36 @@
+# Generated by Django 3.1.8 on 2021-05-29 16:13
+
+from django.db import migrations
+
+
+def forward(apps, schema_editor):
+    EnterpriseEventDefinition = apps.get_model("ee", "EnterpriseEventDefinition")
+    EnterprisePropertyDefinition = apps.get_model("ee", "EnterprisePropertyDefinition")
+
+    EventDefinition = apps.get_model("posthog", "EventDefinition")
+    PropertyDefinition = apps.get_model("posthog", "PropertyDefinition")
+
+    for event in EventDefinition.objects.all():
+        enterprise_eventdef = EnterpriseEventDefinition(eventdefinition_ptr_id=event.id)
```

Comment:
We are downcasting here because doing EnterpriseEventDefinition.objects.create will create a parent (EventDefinition) copy.

Reply:
(which we don't want because we're backfilling the table)

Reply:
we not doing this no more